user/topgrade/template.py

```diff
@@ -0,0 +1,31 @@
+pkgname = "topgrade"
+pkgver = "15.0.0"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo-auditable"]
+pkgdesc = "System updater written in Rust"
```

Comment:
like i said in irc, no "written in <lang>"

Reply:
Fair!